src/contracts.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
This file is not in the right place. Additionally no need for the extra `CONTRACTS_ADDRESSES` - return the object with addresses directly.